src/gc/env/gcenv.ee.h

```diff
@@ -85,6 +85,21 @@ class GCToEEInterface
     static void WalkAsyncPinnedForPromotion(Object* object, ScanContext* sc, promote_func* callback);
     static void WalkAsyncPinned(Object* object, void* context, void(*callback)(Object*, Object*, void*));
     static IGCToCLREventSink* EventSink();
+
+    static bool CanEnableGCNumaAware();
+    static bool GetNumaProcessorNodeEx(PPROCESSOR_NUMBER proc_no, uint16_t *node_no);
```

Comment:
These are OS abstraction. GCToOSInterface would be more appropriate place for them.